frontend/pages/about/our-team.js

```diff
@@ -10,8 +10,16 @@ import LoadingScreen from "../../components/LoadingScreen";
 import axios from "axios";
 import humps from "humps";
 import { execToClassName, marks, valueToYear } from "../../data/TeamData";
+import { loadExecs } from "../../lib/api";
 
-export default function OurTeam() {
+export async function getStaticProps() {
```

Comment:
Same as [this](https://github.com/unswwit/backend/pull/28#discussion_r962294170)